Pod/Classes/WPMediaCollectionViewController.m

```diff
@@ -351,6 +329,54 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
     return cell;
 }
 
+- (CGSize)collectionView:(UICollectionView *)collectionView
+                  layout:(UICollectionViewLayout *)collectionViewLayout
+referenceSizeForHeaderInSection:(NSInteger)section
+{
+    if ( [self isShowingCaptureCell] && self.showMostRecentFirst)
+    {
+        return CameraPreviewSize;
+    }
+    return CGSizeZero;
+}
+
+- (CGSize)collectionView:(UICollectionView *)collectionView
+                  layout:(UICollectionViewLayout *)collectionViewLayout
+referenceSizeForFooterInSection:(NSInteger)section
+{
+    if ( [self isShowingCaptureCell] && !self.showMostRecentFirst)
+    {
+        return CameraPreviewSize;
+    }
+    return CGSizeZero;
+}
+
+- (UICollectionReusableView *)collectionView:(UICollectionView *)collectionView
+           viewForSupplementaryElementOfKind:(NSString *)kind
+                                 atIndexPath:(NSIndexPath *)indexPath
+{
+    if ((kind == UICollectionElementKindSectionHeader && self.showMostRecentFirst) ||
```

Comment:
I don't think you need the `if` statement here. This should only ever be hit if you have the capture cell enabled, and you return the same view regardless of whether it's the header or the footer? Also, the documentation says you should never return `nil` from this method.